Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.6-alpine
+
+ARG requirements_file=production.txt
```

Comment:
I wouldn't do this. IMO it's better to have just one image across all environments. It's ok if it includes dependencies that are only needed during development (like pytest). The benefit is that you have one artifact shared across all the environments which is more interesting that saving some bits IMO.